config/email_config.py

```
@@ -3,3 +3,4 @@
 USE_EMAIL = os.environ.get('USE_EMAIL', 'TRUE') != 'FALSE'
 EMAIL_DOMAIN = os.environ.get('EMAIL_DOMAIN', 'dsasf.org')
 EMAIL_API_KEY = os.environ.get('EMAIL_API_KEY', None)
+MAILGUN_URL = os.environ.get('MAILGUN_URL', 'https://api.mailgun.net/v3/routes')
```

Would this make more sense to not give it a default so it fails (fast) on boot? I could see ~me~ someone forgetting to inject the proper credentials in prod and not finding out that we're not sending emails but silently failing.